packages/react-dom/src/__tests__/ReactTestUtils-test.js

```diff
@@ -678,6 +665,39 @@ describe('ReactTestUtils', () => {
     );
   });
 
+  it('flushes immediate re-renders with act but advances timers later', () => {
```

Comment:
act doesn't advance timers at all, so this title is a bit wrong.

Reply:
Right, removed that part.